webrender/src/renderer.rs

```diff
@@ -2432,6 +2442,11 @@ impl Renderer {
 
                 let texture = match image.source {
                     ExternalImageSource::NativeTexture(texture_id) => ExternalTexture::new(texture_id, texture_target),
+                    ExternalImageSource::Invalid => {
```

Comment:
can we just skip/ignore this element here instead of trying to add it to the `external_images` list (that will be unlocked later on, which doesn't make sense for a failed lock in the first place)?
It seems hazardous to me to just proceed with TextureId==0 as if it's OK.